demo/src/App.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <div>
+      <h1>Demo of default drawing in leto-modelizer</h1>
+    </div>
+    <div>
+      <svg id='root' width='80vw' height='80vh' style='border: 1px solid black' ></svg>
+    </div>
+  </div>
+</template>
+
+<script>
+import { DefaultDrawer } from 'leto-modelizer-plugin-core'
+import resources from './assets/resources';
+import plugin from './assets/data';
+
+export default {
+  name: 'App',
+  data() {
+    return {
+      drawer: new DefaultDrawer(resources, 'root'),
+    };
+  },
+  mounted() {
+    this.$nextTick(() => {
```

Comment:
Why is it needed ?

Reply:
because, we need that `<svg id='root' (...) ></svg>` exists to draw and we need to wait the nexttick to have all mounted correctly.

Reply:
hum, my bad I retest without it and it's work